contents/handbook/company/teams/extensibility.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Team Extensibility
+sidebar: Handbook
+showTitle: true
+---
+
+## Responsibilities
```

Comment:
(I'm going to use what we come up with here as a template for the other teams)

I'd suggest adding the following sections:
- Mission
- Customer, can be both internal or external
- Output metric, need to discuss but could be library downloads, # of plugins used or whatever. Can be high level for now.

I think the Team Extensibility should figure this out tomorrow. I'll join the meeting tomorrow as an observer and to help you along

Reply:
:+1: and meeting invite sent